### DISCUSSION TITLE
Mermaid is using an old version

### DISCUSSION THREAD

**Discussion by ddarksideofthemoo** [requarks/wiki#5848] [Error / Bug Report] [answered]
Hello! 

The wiki js currently uses an old version (8.8.2). This makes it impossible to use "gitGraph" as well. Is it possible to update the mermaid lib to the current version (9.2.2)?

**Comment by bfig** [comment]
Please! we need to keep this up to date to the latest. It's too good!

**Comment by beatwinthewave** [comment]
I need to use the latest version of mermaid, please. Update the `package.json` file with a version greater than 9.2

**Comment by bekanne** [comment]
It is also not possible to link something with a subgraph.
Example:
```
graph TD
  subgraph test
    node1
  end
  node2 ---> test
```

This is a very important basic functionality.

**Comment by Skywalker1-CZ** [comment]
I don't know if it's the problem with old version, but if I create classic flowchart in 
- https://mermaid.live

And then only copy the flowchart syntax into wiki-js, I see in preview everything good, but when I save it, then I see only:

Syntax error in graph 
mermaid version 8.8.2

Any ideas why? 

This is the example from mermaid.live:
flowchart
A[notifikace] --> B{podminka}
B --> |0| C{podminka2}
B --> |1| G
C --> |0| E[Nelze]
C --> |1| F{Accepted}
F --> |0| E
F --> |1| G{Disabled}
G --> |1| E
G --> |0| H{podminka3}
H --> |Nastaveno| E
H --> |Nenastaveno| D[Lze]


Then i used it like this in wikiJS Markdown
```mermaid
flowchart
A[notifikace] --> B{podminka}
B --> |0| C{podminka2}
B --> |1| G
C --> |0| E[Nelze]
C --> |1| F{Accepted}
F --> |0| E
F --> |1| G{Disabled}
G --> |1| E
G --> |0| H{podminka3}
H --> |Nastaveno| E
H --> |Nenastaveno| D[Lze]
```

In preview is everything okay - you can see in attachment

![Snímek obrazovky z 2023-04-27 11-20-03](https://user-images.githubusercontent.com/66778047/234818997-c581ca20-1024-4833-bd2b-0830d6808dfb.png)


Thanks for any advice,
kind regards

Filip Langer
L2 Linux admin
NETHOST s.r.o.

**Comment by NGPixel** [comment, marked as answer]
The upcoming v3 will use the latest Mermaid version.

**Reply by Siecje** [reply]
When will v3 be available? Can a new version of mermaid be a part of a v2 release?
